aten/src/ATen/native/native_functions.yaml

```diff
@@ -2577,6 +2577,12 @@
   dispatch:
     CPU: per_tensor_affine_qtensor_cpu
 
+- func: qtensor_set_storage_(Tensor(a!) self, Storage storage, int storage_offset, int[] sizes, int[] strides=[]) -> Tensor(a!)
```

Comment:
why do we need to add a separate function? can't we just add a dispatch to `set_` method? It currently doesn't have dispatch but I don't see harm in adding it (just spot-check that generated code makes sense)

Reply:
set_() is wrapping TH legacy method, is there a way to add dispatch there? @gchanan @li-roy

Reply:
I got `RuntimeError: _th_set_ is not implemented for type QuantizedCPUType` when I use `set_1`, 
I think we'll need to turn on TH legacy code generation for QuantizedCPUType in that case, we probably don't need most of the TH legacy code, so maybe a explicit set would be better? or maybe we should remove TH codegen for set_?

Reply:
It might be worth to see whether we can just move _th_set out of TH

Reply:
After #20309, you can just add quantized cpu dispatch to set.